dfc/gcp.go

```diff
@@ -24,6 +24,8 @@ import (
 )
 
 const (
+	gcpApiUrl = "http://storage.googleapis.com"
```

Comment:
gcpAPIURL I think that's the go way, but not sure it looks cool in this case, what about urlGoogleAPI?

Reply:
how about gcsURL? gcs = Google Cloud Storage (after all, this URL is specifically for GCS)

Reply:
sure